### PR TITLE
api: local remote channel balances

### PR DESCRIPTION
**Pull request by JosephGoulden**

### CODE REVIEW

File: kld/src/api/spec.yaml [hunk 1]
```diff
@@ -34,3 +34,39 @@ paths:
...
+      security:
+        - MacaroonAuth: []
+      responses:
+        "200":
+          description: Local-Remote balance returned successfully
+          schema:
+            type: object
+            properties:
+              localBalance:
+                type: integer
```
Comment (yanganto) [inline kld/src/api/spec.yaml:53]:
```suggestion
                type: integer
                minimum: 0
```

File: kld/src/api/spec.yaml [hunk 2]
```diff
@@ -34,3 +34,39 @@ paths:
...
+                description: localBalance
+              remoteBalance:
+                type: integer
```
Comment (yanganto) [inline kld/src/api/spec.yaml:56]:
```suggestion
                type: integer
                minimum: 0
```

File: kld/src/api/spec.yaml [hunk 3]
```diff
@@ -34,3 +34,39 @@ paths:
...
+                description: remoteBalance
+              pendingBalance:
+                type: integer
```
Comment (yanganto) [inline kld/src/api/spec.yaml:59]:
```suggestion
                type: integer
                minimum: 0
```

File: kld/src/api/spec.yaml [hunk 4]
```diff
@@ -34,3 +34,39 @@ paths:
...
+                description: pendingBalance
+              inactiveBalance:
+                type: integer
```
Comment (yanganto) [inline kld/src/api/spec.yaml:62]:
```suggestion
                type: integer
                minimum: 0
```

File: api/src/lib.rs [hunk 5]
```diff
@@ -44,6 +44,8 @@ pub mod routes {
     pub const SET_CHANNEL_FEE: &str = "/v1/channel/setChannelFee";
     /// Close an existing channel with a peer.
     pub const CLOSE_CHANNEL: &str = "/v1/channel/closeChannel/:id";
+    /// Fetch aggregate channel local and remote balances.
+    pub const LOCAL_REMOTE_BALANCE: &str = "/v1/channel/localRemoteBal";
```
Comment (yanganto) [inline api/src/lib.rs:48]: What is the role of abbreviating long words?

`estimateChannelLiquidity` in line 23 did not abbreviate.

Comment (JosephGoulden) [reply api/src/lib.rs:48]: not my design, it's taken from here https://github.com/Ride-The-Lightning/c-lightning-REST/blob/master/controllers/localRemoteBal.js

**Comment by yanganto** [comment]
looks good from my end.  Please merge it. I can not find approve btn in cell phone.  Thanks. :)

**Review by paulmattei — APPROVED** [review]

**Pull request merged**